### PR TITLE
Docs/check package descriptions

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: packages/adapters/readmodel-adapters/resolve-readmodel-lite/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-readmodel-lite",
   "version": "0.20.25",
-  "description": "This package serves as a resolve-query adapter for storing a read model in memory",
+  "description": "An adapter used by resolve-query to store read model state in memory",
```
Comment (dirk-pieterse) [inline packages/adapters/readmodel-adapters/resolve-readmodel-lite/package.json]:
```suggestion
  "description": "An adapter the resolve-query uses to store the read model state",
```

File: packages/adapters/readmodel-adapters/resolve-readmodel-mongo/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-readmodel-mongo",
   "version": "0.20.25",
-  "description": "This package serves as a resolve-query adapter for storing a read model in mongo",
+  "description": "An adapter used by resolve-query to store read model state in MongoDB",
```
Comment (dirk-pieterse) [inline packages/adapters/readmodel-adapters/resolve-readmodel-mongo/package.json]: same here

File: packages/adapters/readmodel-adapters/resolve-readmodel-mysql/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-readmodel-mysql",
   "version": "0.20.25",
-  "description": "This package serves as a resolve-query adapter for storing a read model in mysql",
+  "description": "An adapter used by resolve-query to store read model state in MySQL",
```
Comment (dirk-pieterse) [inline packages/adapters/readmodel-adapters/resolve-readmodel-mysql/package.json]: and here

File: packages/adapters/snapshot-adapters/resolve-snapshot-mysql/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-snapshot-mysql",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for snapshots using a file or in mysql.",
+  "description": "An adapter used to save snapshots to MySQL.",
```
Comment (dirk-pieterse) [inline packages/adapters/snapshot-adapters/resolve-snapshot-mysql/package.json]: "save snapshots to" Why 'to'?

File: packages/adapters/storage-adapters/resolve-storage-lite/package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-storage-lite",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for resolve-storage to store events using a file or in memory.",
+  "description": "An adapter that reSolve uses to store events in a file or in memory.",
```
Comment (dirk-pieterse) [inline packages/adapters/storage-adapters/resolve-storage-lite/package.json]:
```suggestion
  "description": "An adapter reSolve uses to store events in a file or memory.",
```

File: packages/adapters/storage-adapters/resolve-storage-postgresql-serverless/package.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-storage-postgresql-serverless",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for resolve-storage to store events using postgresql serverless.",
+  "description": "An adapter that reSolve uses to store events in PostgreSQL Serverless.",
```
Comment (dirk-pieterse) [inline packages/adapters/storage-adapters/resolve-storage-postgresql-serverless/package.json]:
```suggestion
  "description": "An adapter reSolve uses to store events in PostgreSQL Serverless.",
```

File: packages/adapters/storage-adapters/resolve-storage-mongo/package.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-storage-mongo",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for resolve-storage to store events using MongoDB.",
+  "description": "An adapter that reSolve uses to store events in MongoDB.",
```
Comment (dirk-pieterse) [inline packages/adapters/storage-adapters/resolve-storage-mongo/package.json]:
```suggestion
  "description": "An adapter reSolve uses to store events in MongoDB.",
```

File: packages/adapters/storage-adapters/resolve-storage-mysql/package.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-storage-mysql",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for resolve-storage to store events using mysql.",
+  "description": "An adapter that reSolve uses to store events in MySQL.",
```
Comment (dirk-pieterse) [inline packages/adapters/storage-adapters/resolve-storage-mysql/package.json]: Same change

File: packages/adapters/subscribe-adapters/resolve-subscribe-mqtt/package.json [hunk 9]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-subscribe-mqtt",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for resolve-redux subscribe to events.",
+  "description": "An adapter that resolve-redux uses to subscribe to events via the MQTT protocol.",
```
Comment (dirk-pieterse) [inline packages/adapters/subscribe-adapters/resolve-subscribe-mqtt/package.json]: and here

File: packages/adapters/subscribe-adapters/resolve-subscribe-socket.io/package.json [hunk 10]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-subscribe-socket.io",
   "version": "0.20.25",
-  "description": "This package serves as an adapter for resolve-redux subscribe to events.",
+  "description": "An adapter that resolve-redux uses to subscribe to events via Socket.io.",
```
Comment (dirk-pieterse) [inline packages/adapters/subscribe-adapters/resolve-subscribe-socket.io/package.json]: and here

File: packages/core/resolve-local-event-broker/package.json [hunk 11]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-local-event-broker",
   "version": "0.20.25",
-  "description": "This package includes event broker for local run.",
+  "description": "The reSolve framework's event broker for applications that run on a local machine.",
```
Comment (dirk-pieterse) [inline packages/core/resolve-local-event-broker/package.json]:
```suggestion
  "description": "The reSolve framework's event broker for applications on a local machine.",
```

File: packages/core/resolve-scripts/package.json [hunk 12]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "resolve-scripts",
   "version": "0.20.25",
-  "description": "This package create server with resolve.",
+  "description": "Contains scripts usend to initialize a reSolve server.",
```
Comment (dirk-pieterse) [inline packages/core/resolve-scripts/package.json]: usend?

File: packages/adapters/readmodel-adapters/resolve-readmodel-mongo/package.json (hunk 2 above)
Comment (EugeniyBurmistrov) [reply packages/adapters/readmodel-adapters/resolve-readmodel-mongo/package.json]: done

File: packages/adapters/readmodel-adapters/resolve-readmodel-mysql/package.json (hunk 3 above)
Comment (EugeniyBurmistrov) [reply packages/adapters/readmodel-adapters/resolve-readmodel-mysql/package.json]: done

File: packages/adapters/snapshot-adapters/resolve-snapshot-mysql/package.json (hunk 4 above)
Comment (EugeniyBurmistrov) [reply packages/adapters/snapshot-adapters/resolve-snapshot-mysql/package.json]: changed to 'in'

File: packages/adapters/storage-adapters/resolve-storage-mysql/package.json (hunk 8 above)
Comment (EugeniyBurmistrov) [reply packages/adapters/storage-adapters/resolve-storage-mysql/package.json]: done

File: packages/adapters/subscribe-adapters/resolve-subscribe-mqtt/package.json (hunk 9 above)
Comment (EugeniyBurmistrov) [reply packages/adapters/subscribe-adapters/resolve-subscribe-mqtt/package.json]: done

File: packages/adapters/subscribe-adapters/resolve-subscribe-socket.io/package.json (hunk 10 above)
Comment (EugeniyBurmistrov) [reply packages/adapters/subscribe-adapters/resolve-subscribe-socket.io/package.json]: done

File: packages/core/resolve-scripts/package.json (hunk 12 above)
Comment (EugeniyBurmistrov) [reply packages/core/resolve-scripts/package.json]: fixed

**Pull request merged**